quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/Build.java

```diff
@@ -117,11 +115,7 @@ private void beforeReaugmentationOnWindows() {
     private void cleanTempResources() {
         if (!ProfileManager.getLaunchMode().isDevOrTest()) {
             // only needed for dev/testing purposes
-            try {
-                Files.delete(getHomePath().resolve("quarkus-artifact.properties"));
-            } catch (IOException cause) {
-                throw new RuntimeException("Failed to delete temporary resources", cause);
-            }
+            getHomePath().resolve("quarkus-artifact.properties").toFile().delete();
```

Comment:
do we really not care here when delete returns false? (so we fail to delete this file?)

Reply:
No, we do not. Dealing with that only complicates code, I think.